packages/@initiative.dev/ide/src/ide/app.tsx

```diff
@@ -31,6 +35,11 @@ export function App({ projectId, definitions, formatJsFile }: AppProps) {
   const [locale, setLocale] = useState("");
   const [document, setDocument] = useState<SceneDocument | null>(null);
   const [selectedNode, setSelectedNode] = useState<string | null>(null);
+  const [hidden, setDFI_SV] = useState<boolean>(false);
```

Comment:
State value and setter should have matching names, i.e. `[hidden, setHidden]`.

_hidden_ is not a descriptive name – what exactly is hidden? How about this instead: `const [mainAreaTool, setMainAreaTool] = useState<"stage" | "data-flow-inspector">("stage");`